### ISSUE TITLE
Breaking the Black-box

### ISSUE THREAD

**Issue by udaykiranreddykondreddy** [HydPy/HydPy-meetups#9]
**Title of the talk/workshop**
Breaking the Black-box

**Abstract of the talk/workshop**
With the more complex algorithms like deep neural networks, random forest with 1000s of trees or dense machine learning models we are achieving the desired accuracy with a sacrifice of interpretability. If we are more interested in interpretability, we are sacrificing accuracy. In domains like finance or banking both are needed in justifying a prediction which helps the client and customers to understand why it predicted in that way. so how do we build interpretable machine learning models or explainable artificial intelligence? In this workshop, I will be explaining why it is important to build Interpretables models and how to draw insights from it and how to trust your model and make human to understand them, with the help of available methods.

**Outline** 
- Importance of interpretable machine learning: Understanding Machine Learning Model Interpretation, Importance of Machine Learning Model Interpretation, Criteria for Model Interpretation Methods, Scope of Model Interpretation
- Model interpretation techniques: Accuracy vs. Interpretability trade-off, Model Interpretation Techniques
- Hands-on: We will be using the latest state-of-the-art model interpretation frameworks like ELI5, Skater, Lime and SHAP

**Learning Outcomes:** 
- Getting familiar with frameworks like ELI5, LIME and SHAP 
- Getting familiar with concepts like feature importance, partial dependence plots, interpretability and explanation 

**Duration (including Q&A)**
1 hour

**Level of Audience**
Begineer/Intermediate

**Speaker Bio**
1. I love to give back to the community, with that I started to do blogging about Machine learning on [Instagram](https://instagram.com/learn.machinelearning) which has around 80K followers. I'm working as an ML intern in synopsys as a full time and also as a research associate for MUST research academy as part-time.
2. Company - Synopsys
3. Email - udaykiran.kondreddy@gmail.com
4. Years of Exp - 1 year

**Prerequisites(if any)**
- Basic understanding of machine learning and model building
- Familiarity with Python

**Comment by inovizz** [comment]
@udaykiranreddykondreddy - Would you be OK sharing slides and sample code at-least 4-5 days before the talk?

**Comment by udaykiranreddykondreddy** [comment]
@innoviz yes I can share you before Feb 5th.

**Comment by udaykiranreddykondreddy** [comment]
@inovizz can I do this in the next meetup.

**Comment by inovizz** [comment]
@udaykiranreddykondreddy - Sure, I'm scheduling your talk for 15th March meetup then. Please confirm your availability for the same.

**Comment by udaykiranreddykondreddy** [comment]
@inovizz yes I'm available.

**Comment by inovizz** [comment]
@udaykiranreddykondreddy - apologies for a lot of changes the meetup is scheduled for 21st March. Hope you'd be available?

**Comment by udaykiranreddykondreddy** [comment]
@inovizz yes I'm available on that day.

**Comment by inovizz** [comment]
@udaykiranreddykondreddy - which os you have on your machine? It's regarding connecting to GoTo meeting, please let me know.

**Comment by udaykiranreddykondreddy** [comment]
@inovizz I'm using windows.

**Comment by inovizz** [comment]
Awesome, then please install GoTo meeting client on your machine. We can do a dry run when you are sometime on Thursday or Friday?

**Comment by inovizz** [comment]
Hi @udaykiranreddykondreddy Can you call me at +91-9899838120 or ping me on telegram t.me/inovizz
